cachito/web/models.py

```diff
@@ -1,11 +1,69 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
+from copy import deepcopy
 
 from cachito.web import db
 
 
+request_pkg_manager_table = db.Table(
+    'request_pkg_manager',
+    db.Model.metadata,
+    db.Column('request_id', db.Integer, db.ForeignKey('request.id')),
+    db.Column('pkg_manager_id', db.Integer, db.ForeignKey('pkg_manager.id'))
+)
+
+
 class Request(db.Model):
     """A Cachito user request."""
+    __tablename__ = 'request'
```

Comment:
Optional: This isn't necessary since Flask-SqlAlchemy will automatically generate the table name from the class name in snake case form.

Reply:
This was needed for the association table below. Do you know of a way around it?

Reply:
The Flask-SQLAlchemy docs recommend the following:
```python
request_pkg_manager_table = db.Table(
    'request_pkg_manager',
    db.Column('request_id', db.Integer, db.ForeignKey('request.id')),
    db.Column('pkg_manager_id', db.Integer, db.ForeignKey('pkg_manager.id'))
)
```

You could give that a try. I'm not sure what the `db.Model.metadata` argument provides, but it's not in the docs.

http://flask-sqlalchemy.pocoo.org/2.3/models/#many-to-many-relationships

Reply:
Looks like the `db.Model.metadata` is a parameter for plain SQLAlchemy. It's apparently undocumented for Flask-SQLAlchemy but it doesn't seem to cause any errors. Anyways, thanks for spotting this. Removed it.